packages/api/endpoints/granules.js

```diff
@@ -101,19 +101,12 @@ function _createNewGranuleDateValue() {
  * @returns {Promise<Object>} the promise of express response object
  */
 async function list(req, res) {
+  log.trace(`list query ${JSON.stringify(req.query)}`);
```

Comment:
@Jkovarik mentioned that we generally use log.info, and using log.trace is a change to how we're logging so should be discussed with the team.
see convo https://github.com/nasa/cumulus/pull/3681/files#diff-c8cc016ab2b04a286a25bb2d230e35fe63392b371a4178296f58f6388936d72c
Is there a reason to use log.trace instead of log.info?

Reply:
updated to log.debug